pkg/volume/plugins.go

```diff
@@ -129,6 +129,13 @@ type AttachableVolumePlugin interface {
 	NewDetacher(name string, podUID types.UID) (Detacher, error)
 }
 
+// VolumeLabelerPlugin is an extended interface of VolumePlugin and is used to label persistentvolumes
+// with metadata from external cloud.
```

Comment:
`...is used to label...` is not quite accurate right? Since it's only used to fetch labels.